packages/itmat-ui-react/src/App.tsx

```diff
@@ -17,8 +17,8 @@ class App extends React.Component {
             <ApolloProvider client={client}>
                 <Router>
                     <Switch>
-                        <Route path='/resetPassword/:encryptedEmail/:token' component={ResetPasswordPage}/>
-                        <Route path='/requestResetPassword' component={RequestResetPassword}/>
+                        <Route path="/resetPassword/:encryptedEmail/:token" component={ResetPasswordPage}/>
```

Comment:
why is ui linting rule double quote and the other packages single quote????

Reply:
UI JS is also simple quotes, JSX only is double quote to reflect the HTML convention.